docs/Releasing.md

```diff
@@ -18,7 +18,7 @@ The initial announcement of the upcoming release will be a PR by the release
 manager to the `master` branch with the commit to bump the [VERSION
 file](../VERSION) to _next_ pre-release, e.g. `v0.6.x`, and the commit to update
 the [CHANGELOG](#changelog-updates) "Unreleased" section to `v0.6.0` (and change
-the previous to v0.5.0).
+the previous to `v0.6.0`).
```

Comment:
Actually this change is incorrect, so you should drop it (the previous version relative to unreleased being v0.6.0 should be v0.5.0). Reading through the doc, it's confusing in what it posits to be the "current" state when it uses each of the version numbers and requires a more extensive rewrite. You should just ignore this for now and we'll fix in master later.

Reply:
@CodeMonkeyLeet @andschwa - This was a change from #1613 which is now out of date.  I just took this change.  @andschwa - Could you please clarify?